Jakefile.js

```diff
@@ -239,6 +239,10 @@ var libraryTargets = librarySourceMap.map(function (f) {
     return path.join(builtLocalDirectory, f.target);
 });
 
+var localizationTargets = ["cs", "de", "enu", "es", "fr", "it", "ja", "ko", "pl", "pt-BR", "ru", "tr", "zh-CN", "zh-TW"].map(function (f) {

```

Comment:
`enu` should be `en`

Reply:
that is where the lcg file lives. we do not have an lcl file for it.

Reply:
Added a comment explaining the process